tasks/agent.py

```diff
@@ -103,7 +103,7 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
                 build_exclude.append(ex)
 
     # remove all tags that are only available on debian distributions
-    distname = platform.linux_distribution()[0].lower()
+    distname = distro.id().lower()
```

Comment:
probably has almost no impact, but `id` doesn't seem to be the exact replacement of `platform.linux_distribution()[0]`, see https://distro.readthedocs.io/en/latest/#distro.linux_distribution, can you make sure that if the output doesn't match, the strings we compare this output against are updated?

Reply:
@olivielpeau Yup I did properly checked that but you're right had an error with the redhat entry, 🙇 

I've decided to take a distribution road tour and I've noticed that we probably had issues with CentOS:
```
>>> platform.linux_distribution()[0].lower()
'centos linux'
```
This change will fix this, however, it would mean that previous call of this on CentOS was wrong?!

Reply:
Thanks for checking. This is quite scary, does it mean that we've built the Agent on CentOS with the wrong build tags for a long time? (could you check that?)

Reply:
@olivielpeau OK, we were fine: in our builder image, we are using a CentOS 6 where `platform.linux_distribution()[0].lower()` properly returns `centos`.

The test I did (my comment above) were on CentOS 7, where `centos linux` is returned. It's most likely that the `/etc/redhat-release` file is used by platform implementation:

```
 -----------------------------------------------
| CentOS |        /etc/redhat-release           |
 -----------------------------------------------
|    6   | CentOS release 6.10 (Final)          |
|    7   | CentOS Linux release 7.6.1810 (Core) |
 -----------------------------------------------
```

We don't really need to write this down somewhere (more than this comment) because now that we're using `distro`, we're fine on both CentOS 6 & 7 where `distro.id()` returns `centos`. (confirmed locally)

Reply:
great, thanks for checking! Good to know `distro` gives consistent results as well 👍